constructorio-client/src/main/java/io/constructor/client/ConstructorIO.java

```diff
@@ -133,6 +134,13 @@ public void setApiKey(String apiKey) {
       this.apiKey = apiKey;
     }
 
+    /**
+     * Sets user agent
+     */
+    public void setUserAgent(String userAgent) {
```

Comment:
Should this go into the UserInfo object if we want to track user agent per user?